### ISSUE TITLE
[POC] Store resource in blob storage

### ISSUE THREAD

**Issue by markusdregi** [equinor/ert#1370]
Implement functionality to configure and store a resource in blob storage, pass a reference via the workflow manager and have the compute node pull the data.

This can be done in the first iteration by directly communicating with blob storage. Later this logic should be moving into storage, so that reasonable meta data can be attached and we can have a single point for access of data.

**Comment by dotfloat** [comment]
Shouldn't this come from ERT Storage? We want to persist the user's input data, right?

**Comment by sondreso** [comment]
This is different from the new "ERT storage" in the sense that in this context we are talking about opaque data that ERT can/should not parse. From a users perspective it will probably be part of ERT storage, and the ERT storage database will also need to store pointers to this data probably. So yes this is ERT storage, but also no this is not ERT storage in current master 🙂

**Comment by dotfloat** [comment]
In my mind, ERT Storage includes everything that is meant to be persisted over a long time. This is in contrast with the Ensemble Evaluator which runs for a very limited time.

If we want to persist the user data then we use ERT Storage, regardless if we (ERT) can do anything interesting about it. Since this is the domain of ERT Storage, and we intend the user to communicate with ERT Storage in order to authenticate and then fetch the data from the implementation-defined location.

I'm a bit concerned that if we start out with postgres and blob then we'll get to the situation we're making it hard and inefficient for ourselves in terms of fetching data, like when ERT Storage consisted of a "entity" store and a "blob" store.

Blob store is something we definitely want, but not in Q1 and probably not in Q2 either. 

Basically, my point is that while this issue is something that we'll most likely want to do, it shouldn't be at the top of anyone's priority list.

**Comment by jondequinor** [comment]
I think it should be a part of our cloud work as soon as possible. The opaque data are not strictly ERT's concern; they are space and time consuming; and they are probably going to be extremely costly in a warmer environment like ERT storage. 

Move it to somewhere cheap and glacial early. 

There's also the ERT/Sumo discussion that's probably worth reiterating here.

**Comment by markusdregi** [comment]
Blob work currently represented in: https://github.com/equinor/ert/issues/1426
